presto-main/etc/config.properties

```diff
@@ -43,7 +43,8 @@ plugin.bundles=\
   ../presto-postgresql/pom.xml, \
   ../presto-thrift/pom.xml, \
   ../presto-tpcds/pom.xml, \
-  ../presto-google-sheets/pom.xml
+  ../presto-google-sheets/pom.xml, \
+  ../presto-redshift/pom.xml
```

Comment:
Was there a reason the redshift connector was not included by default in the `presto-main` config file?